actions/checkForAndPlaceOrder.ts

```diff
@@ -57,24 +62,41 @@ const _checkForAndPlaceOrder: ActionFn = async (
   event: Event
 ) => {
   const blockEvent = event as BlockEvent;
-  const { network } = blockEvent;
-  const chainContext = await ChainContext.create(context, network);
-  const { registry } = await init(
+  const { network, blockNumber } = blockEvent;
+  const chainId = toChainId(network);
+  const chainContext = await ChainContext.create(context, chainId);
+  const { registry } = await initContext(
     "checkForAndPlaceOrder",
-    blockEvent.network,
+    chainId,
     context
   );
   const { ownerOrders } = registry;
 
   // enumerate all the owners
   let hasErrors = false;
-  console.log(`[checkForAndPlaceOrder] New Block ${blockEvent.blockNumber}`);
+  let ownerCount = 0;
+  let orderCount = 0;
```

Comment:
Tbh, this was just cause theres a lot of output, and I didn't want to print the params multiple time, so I just added a prefix that makes it a bit easier to see all the logs related to the same order.

I used a counter to not add a long text in the log... 

This is not the SDK, just watch tower, i don't think the people will appreciate a JSDOc in this local vars. Since they are counters, i think with having that in the name would suffice. 

Maybe i missed the point, let me know otherwise